src/DatePicker.js

```diff
@@ -143,9 +145,11 @@ const DatePicker = ({
         to: null,
       });
     }
+    onReset();
     setCalendarVisiblity(false);
   };
   const handleApply = () => {
+    onSubmit();
     setCalendarVisiblity(false);
   };
```

Comment:
I think we still should keep the old onSubmit() and create another function for ur use case.